src/main/kotlin/de/rwth_erstis/discordbot_jvm/Launcher.kt

```diff
@@ -11,4 +12,6 @@ fun main(args: Array<String>) {
     val env = dotenv()
     val token = env["TOKEN"] ?: exitProcess(1)
     val bot = Bot(token)
+    val cmdHandler = CommandHandler(bot)
```

Comment:
The cmdHandler should be added to Bot (Interface) and BotImpl like e.g. the jda.

Reply:
Dann würde ich den CommandHandler aber auch nicht mehr als Singleton machen

Reply:
Ja warum sollte es denn eins sein?